vayesta/ewf/fragment.py

```diff
@@ -493,12 +479,12 @@ def make_fragment_dm2cumulant_energy(self, eris=None, t_as_lambda=False, sym_t2=
         if eris is None:
             eris = self.base.get_eris_array(self.cluster.c_active)
         dm2 = self.make_fragment_dm2cumulant(t_as_lambda=t_as_lambda, sym_t2=sym_t2, approx_cumulant=approx_cumulant,
-                full_shape=False)
```

Comment:
Full shape is a waste of memory for MP2

Reply:
This was just a stopgap to get the energy calculation to work.

Reply:
See below- only temporary